src/modules/dom-menupulator.js

```diff
@@ -0,0 +1,122 @@
+class DomMenupulator {
+  constructor(todos) {
+    DomMenupulator.todoList = todos;
+  }
+
+  static todoList;
+
+  static editing = false;
+
+  static displayTodos(domEle, todos) {
+    domEle.innerHTML = '';
+    todos.forEach((todo) => {
+      const list = this.draw(todo);
+      domEle.appendChild(list);
+    });
+  }
+
+  static draw(todo) {
+    this.li = document.createElement('li');
+    this.li.classList.add('todo-list-item');
+
+    this.li.addEventListener('click', this.enableEditing.bind({
+      li: this.li,
+      todo,
+    }));
+
+    this.infoDiv = document.createElement('div');
+    this.infoDiv.classList.add('item-info');
+
+    // Check icon
+    this.checkIcon = document.createElement('span');
+    this.checkIcon.className = 'check-icon';
+    this.sqr = document.createElement('i');
+    this.sqr.className = 'fa-regular fa-square';
+    this.checkIcon.appendChild(this.sqr);
```

Comment:
- [ ] when trying to check the box it doesn't work, instead, the user gets the edit option.